src/js/entities-service/form-responses.js

```diff
@@ -23,6 +24,14 @@ const Entity = BaseEntity.extend({
     }, { filter: {} });
     return fetcher(`/api/patients/${ patientId }/form-responses/latest`, { data }).then(handleJSON);
   },
+  fetchLatestSubmissionByAction(actionId, filter) {
+    const data = reduce(filter, (filters, value, key) => {
+      if (!value) return filters;
+      filters.filter[key] = value;
+      return filters;
+    }, { filter: {} });
+    return fetcher(`/api/actions/${ actionId }/form-responses/latest`, { data }).then(handleJSON);
```

Comment:
This API endpoint doesn't exist on the backend yet. Shortcut story: https://app.shortcut.com/roundingwell/story/38123/.

I copied the `filter` stuff over from the `fetchSubmission`. I assumed we'd want that, but wasn't 100% sure.

Reply:
Currently missing coverage for the `filter` portion.